src/tools/ConsoleMonitor/main.cpp

```diff
@@ -0,0 +1,187 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+// This application allows you to monitor the text buffer contents of ConPTY.
+// All you need to do is run this application in Windows Terminal and it will pop up a window.
+// At the time of writing the implementation is rudimentary. It has no support for wide glyphs and is very slow.

```

Comment:
I considered fixing both these issues, but I felt like it's unnecessary until such functionality is actually needed (and at that point anyone can feel free to contribute such improvements). I only wrote this to quickly test my code after all. 🙂

Reply:
Actually... Thinking about it, I realize that I might want to keep this as a draft until I've actually finished my text reflow PR. That would ensure that this application actually contains everything that I needed to test my PR.

We can of course also merge it earlier if anyone would like.